assets/app/scripts/controllers/services.js

```diff
@@ -24,7 +25,24 @@ angular.module('openshiftConsole')
       $scope.emptyMessage = "No services to show";
       updateFilterWarning();
       console.log("services (subscribe)", $scope.unfilteredServices);
-    }));    
+    }));
+
+    watches.push(DataService.watch("routes", $scope, function(routes){
+        $scope.routes = sortRoutes(routes._data);
```

Comment:
Actually, if `serviceName` will always be present on a route (at least as an empty string), you should be able to just do `$scope.routesByService = routes.by("serviceName");` and get rid of the `sortRoutes` function. (@jwforres, check me on that)